### PR TITLE
Rule application view features.

### PR DESCRIPTION
**Pull request by SpringVaS**
Caught Rule application error in preview mode. Added keyboard shortcuts to timeline view. Reorganized RuleGrid panel.

### CODE REVIEW

File: modules/fxgui/src/edu/kit/iti/algover/MainController.java [hunk 1]
```diff
@@ -90,6 +90,11 @@
     // REVIEW: Would <String> not be more appropriate?
     private final CostumBreadCrumbBar<Object> breadCrumbBar;
 
+    /**
+        Flag whether The displayed terms are just a prevew to avoid rule application consideration
```
Comment (JonasKlamroth) [inline modules/fxgui/src/edu/kit/iti/algover/MainController.java]: typo "preview"

File: modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java [hunk 2]
```diff
@@ -44,25 +44,31 @@ public TimelineLayout(Node... nodes) {
...
                     event.consume();
                 } else if (event.getCode() == KeyCode.LEFT) {
                     moveFrameLeft();
                     event.consume();
+                } else if (event.getCode() == KeyCode.DIGIT1) {
+                    goToView(0);
+                } else if (event.getCode() == KeyCode.DIGIT2) {
+                    goToView(1);
+                } else if (event.getCode() == KeyCode.DIGIT3) {
+                    goToView(2);
```
Comment (JonasKlamroth) [inline modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java]: seems to work fine for me however if a text pane is selected the pressed digit is still printed. Would be nice to catch that

Comment (JonasKlamroth) [reply modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java]: additionally this leaves the text-views (at least the dafny-view) broken

File: modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java [hunk 3]
```diff
@@ -104,6 +112,8 @@ public boolean moveFrameRight() {
         framePosition++;
         updateFrame();
 
+        System.out.println("new Frame pos " + framePosition);
```
Comment (JonasKlamroth) [inline modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java]: as Matthias already mentioned: pls remove or comment out printlns for master

File: modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java (hunk 2 above)
Comment (SpringVaS) [reply modules/fxgui/src/edu/kit/iti/algover/timeline/TimelineLayout.java]: I am reviewing this problem. This is also something about OS interface design, after all we are using the default listeners from the editor components. Naturally, when the "Alt"-key is pressed text is still entered. When using the "Ctrl"-key the arrow keys function as word wise navigation through the editor text.

**Comment by JonasKlamroth** [comment]
i think this should be merged. maybe you can review the conflict and then merge this PR @SpringVaS

**Pull request closed without merging**